core/gui/src/app/dashboard/component/dashboard.component.ts

```diff
@@ -66,7 +66,7 @@ export class DashboardComponent implements OnInit {
             document.cookie = `flarum_remember=${response.token};path=/`;
           },
           error: (err: unknown) => {
-            if ((err as HttpErrorResponse).status == 404) {
+            if ((err as HttpErrorResponse).status == 404 || 500) {
```

Comment:
I think this should be `[404, 500].includes((err as HttpErrorResponse).status)`

Reply:
Fixed.